MiniumApps/launcher3/src/main/AndroidManifest.xml

```diff
@@ -79,7 +79,8 @@
     <uses-permission android:name="android.permission.USES_POLICY_FORCE_LOCK" />
     <uses-permission android:name="android.permission.SYSTEM_ALERT_WINDOW"/>
     <uses-permission android:name="android.permission.FOREGROUND_SERVICE"/>
-
+    <uses-permission android:name="android.permission.QUERY_ALL_PACKAGES"
+                     tools:ignore="QueryAllPackagesPermission"/>
```

Comment:
this permission has nothing to do. still, we can access all installed applications without this permission unless you specify a query filter in the manifest file

Reply:
When this permission isn't specified only subset of all app is available for email category and some others.

Without permission
https://user-images.githubusercontent.com/5735844/128639165-4738c8a0-b53a-4a08-a5d0-dad78517fa7e.mp4

With permission
https://user-images.githubusercontent.com/5735844/128639175-80b2bed8-2594-4ad2-808f-d31d7c0f4330.mp4